### ISSUE TITLE
Allow using it with android

### ISSUE THREAD

**Issue by seniordevel** [CarlosAMolina/checkIframe#3]
Please allow using this addon with android.
It should be compatible without modifying its code.

Just add some lines to manifest.json:

"browser_specific_settings": {
  "gecko_android": {
    "id": "your addon id",
    
  }

**Comment by CarlosAMolina** [comment]
Hello @seniordevel,
Thank you for the suggestion. I will review it.
Regards.

**Comment by CarlosAMolina** [comment]
Hello @seniordevel,

Version 1.5.0 is now available and allows you to install the add-on on Firefox for Android devices.

Sorry for the delay in implementing this feature.

Please note that the Android version does not update the add-on icon color. You have more details in the documentation <https://cmoli.es/projects/check-iframe/how-to-use/work-in-android-device.html>.

I close this topic, do not hesitate to contact me for any questions.

Thank you for you interest in this add-on!

Regards,
Carlos.

**Comment by seniordevel** [comment]
Sorry, but does not work for me.

With Firefox desktop edition, I have set the following CSS code in usercontent.css:

iframe{border-style: solid !important; border-widh:2px !important; border-color:#00ff00 !important; overflow: auto !important;}

This should beware me from malicious addons maninipulating Google search results as happened to me in the past.

As shown at your images -advertising the CheckIframe addon- your addon should expose any iframe with a red border. 

This does not happen here.
Tested at:

https://www.w3schools.com/html/html_iframe.asp
